Defs/ThingDefs_Misc/Weapons_RangedNeolithic.xml

```diff
@@ -28,6 +34,7 @@
 				<compClass>CompEquippable</compClass>
 			</li>
 		</comps>
+		<relicChance>1</relicChance>
```

Comment:
I don't think it's desirable to have the Pilum as a relic, since they are a consumable, and I'm not sure if there are any others that use this base?